### PR TITLE
3523-0

### PR DESCRIPTION
**Pull request by 40grivenprog**
## Номер
3523

## Номер задания
1

## Ссылка на видео с демо
https://www.youtube.com/watch?v=SzNVWg0KYc0

## Комментарии
Хотел сделать то-то, но не успел и не понимаю.

### CODE REVIEW

**Comment by Vadser** [comment]
why do you have duplicated commits

**Comment by 40grivenprog** [comment]
> why do you have duplicated commits

cause i fixed rubocop in this file a lot of times

File: 3523/0/about_dice_project.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+# Implement a DiceSet Class here:
+# :reek:Attribute
+# :reek:InstanceVariableAssumption:
+# :reek:UncommunicativeVariableName
+class DiceSet
+  attr_accessor :values
+  def roll(num)
+    @values = []
+    (1..num).each do |i|
```
Comment (Vadser) [inline 3523/0/about_dice_project.rb]: What about using each_with_object instead each

File: 3523/0/triangle.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# :reek:all
+# rubocop:disable Naming/UncommunicativeMethodParamName
+# rubocop:disable Style/NumericPredicate
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
```
Comment (Vadser) [inline 3523/0/triangle.rb]: This file you write by yourself, so you can't disable any errors here, please fix all this errors

File: 3523/0/triangle.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+def triangle(a, b, c)
```
Comment (Vadser) [inline 3523/0/triangle.rb]: Please refactor this method, it is hard readable, also don't use one letter as variable name

File: 3523/0/about_scoring_project.rb [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+# Your goal is to write the score method.
+
+def score(dice)
+  sum(dice)
+end
+
+# :reek:NilCheck
+# :reek:TooManyStatements
+# :reek:UncommunicativeVariableName:
+def sum(arr)
```
Comment (Vadser) [inline 3523/0/about_scoring_project.rb]: You can't disable errors for methods which you wrote by yourself, please fix them, also refactor this methods, they are hard readable, also change variable naming in this methods

**Pull request closed without merging**